client/asset/eth/eth.go

```diff
@@ -379,9 +379,9 @@ func (eth *ExchangeWallet) balance() (*asset.Balance, error) {
 		return nil, err
 	}
 
-	locked := eth.locked + eth.redemptionReserve + dexeth.WeiToGwei(bal.PendingOut)
+	locked := eth.locked + eth.redemptionReserve
 	return &asset.Balance{
-		Available: dexeth.WeiToGwei(bal.Current) - locked,
+		Available: dexeth.WeiToGwei(bal.Current) - locked - dexeth.WeiToGwei(bal.PendingOut),
```

Comment:
Looks good as per discussion in https://github.com/decred/dcrdex/issues/1460#issuecomment-1029394466